test/misc.c

```diff
@@ -0,0 +1,25 @@
+#include "greatest.h"
+
+#ifndef VERSION
+#define VERSION ""
```

Comment:
Why cause a fatal error in `dunst.c` if VERSION is not set but gracefully handle it here?